Xamarin.Forms.Platform.iOS/Renderers/DatePickerRenderer.cs

```diff
@@ -160,7 +162,31 @@ void UpdateDateFromModel(bool animate)
 			if (_picker.Date.ToDateTime().Date != Element.Date.Date)
 				_picker.SetDate(Element.Date.ToNSDate(), animate);
 
-			Control.Text = Element.Date.ToString(Element.Format);
+			//can't use Element.Format because it won't display the correct format if the region and language are set differently
+			if (Element.Format.Equals("") || Element.Format.Equals("d") || Element.Format.Equals("D"))
```

Comment:
Do you think we should use something like `string.IsNullOrWhitespace` instead of `.Equals("")`?